airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -265,27 +288,31 @@ def _upload_to_gcs(self, files_to_upload):
                         tmp_file.get('file_handle').name,
                         mime_type=tmp_file.get('file_mime_type'))
 
-    @staticmethod
-    def _convert_types(schema, col_type_dict, row):
+    @classmethod
+    def _convert_types(cls, schema, col_type_dict, row):
+        return [
+            cls._convert_type(value, col_type_dict.get(name))
+            for name, value in zip(schema, row)
+        ]
+
+    @classmethod
+    def _convert_type(cls, value, schema_type):
```

Comment:
The documentation for this conversion is good, but in my opinion we should always document the parameters as well + to have a cleaner sphinx generated documentation. :)